TeamSnapSDK/SDK/Cache/TSPCache.m

```diff
@@ -22,16 +22,26 @@ +(void)setCacheRootPath:(NSURL *)rootPath {
 
 +(NSURL *)cacheRootPath {
     if (!_rootPath) {
-        NSString *directory = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) lastObject];
+        NSString *directory = [NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES) lastObject];
         _rootPath = [[NSURL fileURLWithPath:directory isDirectory:YES] URLByAppendingPathComponent:@"teamsnap"];
     }
     if (![[self fileManager] fileExistsAtPath:[_rootPath path]]) {
         [[self fileManager] createDirectoryAtPath:[_rootPath path] withIntermediateDirectories:YES attributes:nil error:nil];
+        [self markDirectoryForNoBackup:_rootPath];
```

Comment:
Interesting... so then we just have to decide what we want to do in this situation. I don't think I would consider us qualified for a case similar to "file necessary to function, such as offline". These files are only a cache for convenience and the network is still required for the app to function. I don't think we should mark them as unpurgeable simple because we don't manage purging them ourselves and we download from the Internet subsequently regardless.